# MattGreen-PhD

### Welcome, here you will find code related to the projects outlined below in reverse chronological order. Please don't hesitate to reach out with any questions. 

### **Email:** ##Mattgreen901@gmail.com





#### Single-cell sequencing analysis of Gad2-positive neurons from the lateral habenula

###### This repository contains R code that performs the following analyses
######   - Single-cell QC analyses
######   - Clustering analyses using PCA and UMAP projections
######   - Differential expression to search for relevant genes between clusters
######   - GO analyses using TopGO
######   - Motif enrichment analyses using RcisTarget


#### Simulating mutations in the H1 family of histone genes and calculating predicted resulting protein charge
###### This repository contains R code that simulations single base pair insertions or deletions at each bp in each gene of the histone H1 family. It also calculates the predicted charge of the resulting protein from each mutation and plots mutation vs predicted charge.


#### Generating allele-specific read counts in order to estimate genome-wide transcriptional bursting kinetics
###### This repository contains code to conduct the pre-processing involved to generate allele-specific read counts from single cell sequencing data using C. 
###### It also contains code using R to conduct the estimation of transcriptional bursting kinetics using the read counts under the pipeline produced in the Jiang et al., 2017 paper *SCALE: modeling allele-specific gene expression by single-cell RNA sequencing.* 


#### Code from *Automated Live-Cell Imaging of Synapses in Rat and Human Neuronal Cultures*. Green et al., 2019. 
###### This repository was made by Thomas Pengo and myself and contains pipelines for the automated analysis of images of fluorescently labeled synapses in live neurons.  


<!---
MattGreen-PhD/MattGreen-PhD is a ✨ special ✨ repository because its `README.md` (this file) appears on your GitHub profile.
You can click the Preview link to take a look at your changes.
--->
